src/Core/Module/ModuleRepository.php

```diff
@@ -378,4 +381,24 @@ protected function enrichModuleAttributesFromHook(Module $module): ModuleInterfa
 
         return $module;
     }
+
+    /**
+     * Filter modules if the current employee has the right to see it
+     */
+    protected function filterModulesFromPermissions(ModuleCollection $modules): ModuleCollection
+    {
+        foreach ($modules as $key => $module) {
+            $moduleName = $module->getAttributes()->get('name');
+            if ($this->moduleDataProvider->isInstalled($moduleName)) {
+                if (!$this->moduleDataProvider->can(
+                    ModulePermission::VIEW,
+                    $moduleName
+                )) {
+                    unset($modules[$key]);
+                }
+            }
```

Comment:
```suggestion
            if ($this->moduleDataProvider->isInstalled($moduleName) && !$this->moduleDataProvider->can(
                ModulePermission::VIEW,
                $moduleName
            )) {
                unset($modules[$key]);
            }
```
No need for nesting. Right?

Reply:
Fixed.